tests/layer_tests/common/tflite_layer_test_class.py

```diff
@@ -30,13 +30,20 @@ def produce_tflite_model(self, framework_model, save_path):
             input_tensors = get_tensors_from_graph(g, self.inputs)
             output_tensors = get_tensors_from_graph(g, self.outputs)
 
-        tflite_model = tf.compat.v1.lite.TFLiteConverter(framework_model,
+        converter = tf.compat.v1.lite.TFLiteConverter(framework_model,
                                                          input_tensors=input_tensors,
-                                                         output_tensors=output_tensors).convert()
+                                                         output_tensors=output_tensors)
+        converter.target_spec.supported_ops = [
+          tf.lite.OpsSet.TFLITE_BUILTINS, # enable TensorFlow Lite ops.
+          tf.lite.OpsSet.SELECT_TF_OPS # enable TensorFlow ops.
+        ]
+        tflite_model = converter.convert()
 
         tflite_model_path = os.path.join(os.path.dirname(save_path), 'model.tflite')
         with tf.io.gfile.GFile(tflite_model_path, 'wb') as f:
             f.write(tflite_model)
+        with tf.io.gfile.GFile('E:/openvino/ov_tflite/tests/layer_tests/tensorflow_lite_tests/densify.tflite', 'wb') as f:
+            f.write(tflite_model)
```

Comment:
please remove hard-coded path

Reply:
It is part of a future layer test, shouldn't be in this PR. I've removed it.